MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -503,17 +500,6 @@ public string GetInputArgument(EncodingJobInfo state, EncodingOptions encodingOp
                 && state.SubtitleDeliveryMethod == SubtitleDeliveryMethod.Encode
                 && state.SubtitleStream.IsExternal && !state.SubtitleStream.IsTextSubtitleStream)
             {
-                if (state.VideoStream != null && state.VideoStream.Width.HasValue)
-                {
-                    // This is hacky but not sure how to get the exact subtitle resolution
-                    int height = Convert.ToInt32(state.VideoStream.Width.Value / 16.0 * 9.0);
-
-                    arg.Append(" -canvas_size ")
-                        .Append(state.VideoStream.Width.Value.ToString(CultureInfo.InvariantCulture))
-                        .Append(':')
-                        .Append(height.ToString(CultureInfo.InvariantCulture));
-                }
-
```

Comment:
Since we have manually set the graphical subtitle size in `GetGraphicalSubtitleParam()` according to the video size, this area will not work. And `libx264` with `sub2video` can adaptive video size and graphic subtitle size.